packages/vue/src/components/result/ReactiveList.jsx

```diff
@@ -114,6 +114,7 @@ const ReactiveList = {
 		pagination: VueTypes.bool.def(false),
 		paginationAt: types.paginationAt.def('bottom'),
 		react: types.react,
+		scrollOnChange: VueTypes.bool.def(false),
```

Comment:
Hey thanks for the PR 💯 

default value should be `true` like it is in React version